tests/docs/sandbox/generic/authentication.rst

```diff
@@ -1,8 +1,8 @@
 Authentication Tests
 ====================
 
-401 - Unauthenticated
```

Comment:
A 401 is where the user has sent a non-authenticated request, 403 is for unauthorized

Authentication is the determination that the request is coming from an authentic identity.

Authorization is the determination that an authentic identity has access to the resource/operation being requested.